src/domain/usecases/SendNotificationDataValuesUseCase.ts

```diff
@@ -362,14 +362,16 @@ export class SendNotificationDataValuesUseCase {
     }
 
     private async sendEmailToUsersInGroup(users: User[], subject: string, body: string): Async<void> {
-        await this.notificationsRepository.send({
-            attachments: [],
-            subject,
-            body: {
-                contents: body,
-                type: "html",
-            },
-            recipients: users.map(user => user.email),
+        await promiseMap(users, async user => {
+            await this.notificationsRepository.send({
+                attachments: [],
+                subject,
+                body: {
+                    contents: body,
+                    type: "html",
+                },
+                recipients: [user.email],
+            });
```

Comment:
So finally sending N emails instead of BCC, right?

Reply:
Yep, bcc is still including info from other users (gmail and outlook, not sure about other mail clients).

![image](https://github.com/EyeSeeTea/d2-tools/assets/461124/e1829703-0f2d-4709-885a-3022ec2ba515)

Reply:
Oh, good to know, it looks like a bug in nodemailer. Sending N emails seems also fine for this use case.